cloud-run/src/environment-args.js

```diff
@@ -6,6 +6,7 @@ const createEnvironmentArgs = (environment, projectId) => {
     args.push(`${name}=${value}`);
   });
   args.push(`SERVICE_PROJECT_ID=${projectId}`);
+  args.push(`SERVICE_ENVIRONMENT=${projectId.split('-')[1]}`);
```

Comment:
This will break if a project name includes a hyphen. It would be safer to do something like:

```
args.push(`SERVICE_ENVIRONMENT=${projectId.includes('-staging-') ? 'staging' : 'prod'}`);
```

Reply:
You're right, i thought the project id followed the convention of name-env-id, will resolve this now with your solution 👍